floris/tools/optimization/scipy/yaw_wind_rose.py

```diff
@@ -584,15 +668,15 @@ def calc_baseline_power(self):
                     )
 
                 # calculate baseline power
-                self.fi.calculate_wake(yaw_angles=0.0)
+                self.fi.calculate_wake(yaw_angles=self.x0, no_wake=False)
```

Comment:
This is tricky. What if we fix a particular turbine to `10.0` deg yaw for all conditions, e.g., mimicking a wind vane error? Then the baseline situation may be better than the optimal solution. I'd like to see x0 as the baseline situation. However, I completely get your point too. TBD.

Reply:
Yeah, I see your point about if we want to fix certain turbines' yaw offsets to non-zero values. So it's not fair to force the yaw angles to 0 for baseline calculations. But I think x0 should stay as the initial condition for optimization purposes, since there could be times when we have a non-zero initial guess we want to use for finding the optimal yaw offsets, but want to assume all turbines are aligned in the baseline case. 

One solution could be to use whatever the current yaw angles of the floris object are when the optimization method is called as the "baseline" yaw angles (which could be non-zero). This way we can independently specify the baseline offsets, the initial conditions for optimization, and the bounds for optimization. And we wouldn't need to add another argument to the class. Or we could add another argument to make it clear. Let me know what you think!

Reply:
Great suggestion Eric! I created an additional variable called self.x_baseline which contains the yaw angles of the floris object as it is inserted into the yaw optimization class. These yaw angles are used to now calculate the baseline power production/initial_farm_power. The user can still insert a different value for x0 as the initial condition for the optimization algorithm. I have added a print statement to raise a warning when x0 differs from x_baseline to inform the user that this is the case, and to confirm that they are doing this on purpose.